core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java

```diff
@@ -98,4 +99,12 @@ public void testFixedRandomUniqueness() {
         assertTrue(split[0] != split[1]);
     }
 
+
+    @Test
+    public  void testEnvPropertyReplace(){
+        String text = "${SYSTEM.ENV:PWD}";
```

Comment:
May be an assertion with a know env value of `JAVA_HOME` types will help

Reply:
Yeah, `JAVA_HOME` can be used, but how can we verify it?
In the case of `PWD`, we can verify it by fetching the current directory from the Paths API

Reply:
right, that's ok. Understood.
Then just add
`assertThat(....)` bit to the test 👍

Reply:
Yeah, Forgot the assertion part :man_facepalming:

Reply:
I have modified the test to make more sense. I guess it should be okay. Feel free to throw your suggestions.